scripts/map.js

```diff
@@ -100,6 +100,7 @@ var loadZones = function (geojson) {
 
 	// Turn on federal/state land by default
 	$('input[name="Overlay"][value="fs"]').prop('checked', true)
+	$('input[name="Overlay"][value="kauai"]').prop('checked', true)
```

Comment:
Can remove this as well

Reply:
You can probably override adding Kauai with :

```javascript
overlays["kauai"].addTo(map)
```

Reply:
Instead of the hidden checkbox, got it.